src/platform/webos/CHIPLinuxStorage.cpp

```diff
@@ -0,0 +1,321 @@
+/*
+ *
+ *    Copyright (c) 2020 Project CHIP Authors
```

Comment:
Is this a copy of linux? Name says "CHIPLinuxStorage" but platform is webos.

Reply:
@andy31415 
As I'm not ready for using the own storage implementation for webOS, let me remove this and re-use the one from Linux.
Thank you for your review.